#
# This file is part of the LibreOffice project.
#
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.
#
--- a/xmlsecurity/qa/unit/signing/signing.cxx
+++ b/xmlsecurity/qa/unit/signing/signing.cxx
@@ -45,6 +45,51 @@
 #include <documentsignaturemanager.hxx>
 #include <certificate.hxx>
 
+// org.freedesktop.Sdk//1.6 contains "gpgconf (GnuPG) 2.1.14-unknown" and fails
+// CppunitTest_xmlsecurity_signing with
+//
+// [...]
+// SigningTest::testODFBrokenStreamGPG finished in: 257ms
+// gpgconf: error removing '/run/user/1000/gnupg/d.ddj1dwjpi9iofkpjnicqebif': Directory not empty
+// gpgconf: fatal error (exit status 1)
+// /run/build/libreoffice/xmlsecurity/qa/unit/signing/signing.cxx:235:SigningTest::testODFBrokenDsigGPG
+// tearDown() failed
+// - equality assertion failed
+// - Expected: 0
+// - Actual  : 256
+// - LD_LIBRARY_PATH=/app/lib:/usr/lib/sdk/gcc7/lib:/usr/lib/GL/nvidia-410-66/lib:/usr/lib/ffmpeg/lib GNUPGHOME=/run/build/libreoffice/workdir/CppunitTest/xmlsecurity_signing.test.user /usr/bin/gpgconf --remove-socketdir
+//
+// SigningTest::testODFBrokenDsigGPG finished in: 1265ms
+// gpgconf: error removing '/run/user/1000/gnupg/d.ddj1dwjpi9iofkpjnicqebif': Directory not empty
+// gpgconf: fatal error (exit status 1)
+// /run/build/libreoffice/xmlsecurity/qa/unit/signing/signing.cxx:235:SigningTest::testODFEncryptedGPG
+// tearDown() failed
+// - equality assertion failed
+// - Expected: 0
+// - Actual  : 256
+// - LD_LIBRARY_PATH=/app/lib:/usr/lib/sdk/gcc7/lib:/usr/lib/GL/nvidia-410-66/lib:/usr/lib/ffmpeg/lib GNUPGHOME=/run/build/libreoffice/workdir/CppunitTest/xmlsecurity_signing.test.user /usr/bin/gpgconf --remove-socketdir
+//
+// SigningTest::testODFEncryptedGPG finished in: 447ms
+// signing.cxx:235:Assertion
+// Test name: SigningTest::testODFBrokenDsigGPG
+// tearDown() failed
+// - equality assertion failed
+// - Expected: 0
+// - Actual  : 256
+// - LD_LIBRARY_PATH=/app/lib:/usr/lib/sdk/gcc7/lib:/usr/lib/GL/nvidia-410-66/lib:/usr/lib/ffmpeg/lib GNUPGHOME=/run/build/libreoffice/workdir/CppunitTest/xmlsecurity_signing.test.user /usr/bin/gpgconf --remove-socketdir
+//
+// signing.cxx:235:Assertion
+// Test name: SigningTest::testODFEncryptedGPG
+// tearDown() failed
+// - equality assertion failed
+// - Expected: 0
+// - Actual  : 256
+// - LD_LIBRARY_PATH=/app/lib:/usr/lib/sdk/gcc7/lib:/usr/lib/GL/nvidia-410-66/lib:/usr/lib/ffmpeg/lib GNUPGHOME=/run/build/libreoffice/workdir/CppunitTest/xmlsecurity_signing.test.user /usr/bin/gpgconf --remove-socketdir
+//
+// Failures !!!
+// Run: 28   Failure total: 2   Failures: 2   Errors: 0
+#undef HAVE_GPGCONF_SOCKETDIR
+
 using namespace com::sun::star;
 
 namespace
